nfc.py

```diff
@@ -60,6 +62,33 @@ def find(cls):
 	logging.basicConfig(level=logging.DEBUG)
 	logger = logging.getLogger(__name__)
 
+	logger.info("Checking for dependencies...")
+	if "pyusb" not in {pkg.key for pkg in pkg_resources.working_set}:
```

Comment:
As mentioned earlier - use "except" for that.

But there's a "catch" - you should make sure that if someone tries to import classes from nfc.py, it would not print and exit! It must raise ImportError exception as normal.

Maybe extract UsbDeviceMatcher/UsbDeviceFinder into a separate file (without any try/except)? 
Then you can:
```
try:
  import separatefile
except ImportError:
  do something
```

Note that:
- error messages must go to STDERR, not STDOUT,
- failures must cause exit code other than 0 (re-raising an exception might be a great option),
- important problems should be logged as errors (not "info").

Reply:
That will work. Will do it that way instead.